skf/markdown/code_examples/java/1-code_example--CSRF_Token--.md

```diff
@@ -0,0 +1,113 @@
+
+
+CSRF Token 
+-----------
+
+***Example:***
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+/*
+For CSRF tokens we used a separate class outside of the normal controller, since
+it must be re-used on several locations throughout the application
+
+First after a successful validation of a user login, the application must also start a session
+which contains the "cross site request forgery" token.
+
+For generating the token we want to use a secure cryptographic function
+SecureRandom csprng = new SecureRandom();
```

Comment:
@xvass  it's a good practice on to add a more descriptive names on the classes, so if anyone try to copy from this they understand what is the purpose of this variable.